projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -92,6 +92,23 @@ class Contact_Form_Endpoint extends \WP_REST_Posts_Controller {
 	 * @return array Filtered list of supported integrations
 	 */
 	private function get_supported_integrations() {
+		/**
+		 * Filters the list of supported integrations available in Jetpack Forms.
+		 *
+		 * Use this filter to add, modify, or remove integrations. Removing an
+		 * integration here will prevent it from being returned by the REST
+		 * integrations endpoints and from being displayed in the UI.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param array $supported_integrations Associative array of integration
+		 *                                     configurations keyed by slug. Each
+		 *                                     configuration supports the following keys:
+		 *                                     - type (string)                  : 'plugin' or 'service'.
+		 *                                     - file (string|null)             : Plugin file path for plugins; null for services.
+		 *                                     - settings_url (string|null)     : Relative admin URL for settings, or null.
+		 *                                     - marketing_redirect_slug (string|null) : Redirect slug for marketing links, or null.
+		 */
 		return apply_filters( 'jetpack_forms_supported_integrations', $this->supported_integrations );
```

Comment:
This is the filter for filtering whether to hide show integrations. Just above in this file `$this->$supported_integrations` defines an array of supported integrations . Developers can remove (and theoretically add) elements to that array, which will now hide/show them in the integrations panel. 

As an added benefit, in a follow up PR, we can add the default title and subtitle for each integration to `$this->supported_integrations`, and read those from here rather than defining in our frontend JS. That will allow users to filter the title/subtitle for integrations via this filter as well.